frontend/micro-ui/web/micro-ui-internals/packages/react-components/src/atoms/WorkflowActions.js

```diff
@@ -97,7 +97,7 @@ const WorkflowActions = ({ businessService, tenantId, applicationNo, forcedActio
     //send appropriate states over
     
     if(bsEstimate === businessService && action?.action === "RE-SUBMIT"){
```

Comment:
can you remove the url from this comp and use a callback function like **editCallback**

Reply:
added the changes, Please verify